### ISSUE TITLE
The part of the code that calculate mu and sigma, which is is based on the distribution of Mt and Mt-1 ?

### ISSUE THREAD

**Issue by zhe-juanz** [mr-eggplant/adaxpert0#3]
Hello,  your work is really interesting !
Uhhh,  I’m a freshman. According to the code, we can know that the WD distance is calculated by "calculate_distribution_distance". IHowever, it doesn’t seem to find the part of the code that calculate mu and sigma, which is is based on the distribution of Mt and Mt-1, could you please point it out? 
Thx!

**Comment by mr-eggplant** [comment]
Hi, thank you for your interest!

The release code does not consist of "calculating mu and Sigma" because the calculation requires a specific model (well-trained) and datasets (old and new datasets). 

The calculation of W distance is based on M_t and M_{t-1}, which denote the corresponding feature matrix of new and old datasets on old model, respectively. 

To obtain mu and Sigma, u can use the following code:

# get feature embeddings
sub_model.eval()
features_old = []
for batch_idx, (inputs, targets) in enumerate(original_loader):
        inputs, targets = inputs.to(device), targets.to(device)
        with torch.no_grad():
                feature, _ = sub_model(inputs)
                features_old.append(feature)

features_new = []
for batch_idx, (inputs, targets) in enumerate(new_loader):
        inputs, targets = inputs.to(device), targets.to(device)
        with torch.no_grad():
                feature, _ = sub_model(inputs)
                features_new.append(feature)

features_old = torch.cat(features_old)
features_new = torch.cat(features_new)

# compute mean vectors and covariance matrices
old_mu, new_mu = torch.mean(features_old, dim=0), torch.mean(features_new, dim=0)
old_sigma, new_sigma = np.cov(features_old.t().cpu().numpy()), np.cov(features_new.t().cpu().numpy())

Kind regards

**Comment by zhe-juanz** [comment]
Thank you for your kindly reply!
